context-js/src/lib.rs

```diff
@@ -31,7 +31,11 @@ pub fn env_parse(env_vars: js_sys::Object) -> Option<env::parser::CIInfo> {
 }
 
 #[wasm_bindgen]
-pub fn parse_branch_class(value: &str, pr_number: Option<usize>, gitlab_merge_request_event_type: Option<env::parser::GitLabMergeRequestEventType>) -> env::parser::BranchClass {
+pub fn parse_branch_class(
+    value: &str,
+    pr_number: Option<usize>,
+    gitlab_merge_request_event_type: Option<env::parser::GitLabMergeRequestEventType>,
```

Comment:
optional: I really feel like this input should be provider-agnostic or part of a larger struct

(I realize this isn't from your changes)